src/scripts/main.js

```diff
@@ -2,7 +2,29 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('tbody');
 
-// write your code here
+people.forEach(person => {
+  table.insertAdjacentHTML('beforeend', `
+    <tr>
+      <td>
+        ${person.name}
+      </td>
+      <td>
+        ${person.sex === 'f' ? 'Female' : 'Male'}
+      </td>
```

Comment:
probably you have enough space to write it in one line for better readability. Apply for other td's as well 
```suggestion
      <td>${person.name}</td>
      <td>${person.sex === 'f' ? 'Female' : 'Male'}</td>
```